codes/dataops/common.py

```diff
@@ -1,152 +1,150 @@
     if os.path.isfile(keys_cache_file):
-        logger.info('Read lmdb keys from cache: {}'.format(keys_cache_file))
+        logger.info('Read lmdb keys from cache: %s', keys_cache_file)
         keys = pickle.load(open(keys_cache_file, "rb"))
     else:
         with env.begin(write=False) as txn:
-            logger.info('Creating lmdb keys cache: {}'.format(keys_cache_file))
+            logger.info('Creating lmdb keys cache: %s', keys_cache_file)
             keys = [key.decode('ascii') for key, _ in txn.cursor()]
         pickle.dump(keys, open(keys_cache_file, 'wb'))
     paths = sorted([key for key in keys if not key.endswith('.meta')])
     return env, paths
 
 
 def get_image_paths(data_type, dataroot):
-    '''get image path list
-    support lmdb or image files'''
+    """
+    Get image path list.
+    Supports lmdb files or image directories.
+    """
     env, paths = None, None
     if dataroot is not None:
         if data_type == 'lmdb':
             env, paths = _get_paths_from_lmdb(dataroot)
         elif data_type == 'img':
             paths = sorted(_get_paths_from_images(dataroot))
         else:
-            raise NotImplementedError('data_type [{:s}] is not recognized.'.format(data_type))
+            raise NotImplementedError('data_type [%s] is not recognized.' % data_type)
     return env, paths
 
 
-###################### read images ######################
 def _read_lmdb_img(env, key, size=None):
-    '''read image from lmdb with key (w/ and w/o fixed size)
-    size: (C, H, W) tuple'''
+    """
+    Read image from lmdb with key (w/ and w/o fixed size).
+    size: (C, H, W) tuple.
+    """
     with env.begin(write=False) as txn:
         buf = txn.get(key.encode('ascii'))
         buf_meta = txn.get((key + '.meta').encode('ascii')).decode('ascii')
     img_flat = np.frombuffer(buf, dtype=np.uint8)
     if size:
-        C, H, W = size
+        c, h, w = size
     else:
-        H, W, C = [int(s) for s in buf_meta.split(',')]
-    img = img_flat.reshape(H, W, C)
-    return img
+        h, w, c = [int(s) for s in buf_meta.split(',')]
+    image = img_flat.reshape(h, w, c)
+    return image
 
 
 def read_img(env, path, out_nc=3, fix_channels=True, size=None):
-    '''
-        Reads image using cv2 (rawpy if dng), from lmdb bor from a buffer 
-        (path=buffer). Could also use using PIL instead of cv2.
-    Arguments:
-        path: image path or buffer to read
-        out_nc: Desired number of channels
-        fix_channels: changes the images to the desired number of channels
-        size: fixed size to use for lmbg (optional)
-    Output:
-        Numpy HWC, BGR, [0,255] by default 
-    '''
-
-    img = None
-    if env is None or env is 'img':  # img
-        if(path[-3:].lower() == 'dng'): # if image is a DNG
+    """
+    Reads image using cv2 (rawpy if dng), from lmdb bor from a buffer (path=buffer).
+    Could also use PIL instead of cv2.
+    :param path: image path or buffer to read
```

Comment:
"arguments" and "output" separators makes it easier to read, specially on long comments

Reply:
Not sure what you mean here?

Reply:
I believe he's saying to use the other param docstring style